include/event_manager.h

```diff
@@ -30,6 +30,26 @@
 extern "C" {
 #endif
 
+/**
+ * @brief List of bits in event type flags.
+ */
+enum event_type_flags {
+   EVENT_FLAGS_USER_SETTABLE_START,
+   EVENT_FLAGS_INIT_LOG_ENABLE = EVENT_FLAGS_USER_SETTABLE_START,
+   EVENT_FLAGS_TRACEABLE,
+   EVENT_FLAGS_SYSTEM_START, /**< Used to assert the user trying to set flags that should not be changed by the user */
+   EVENT_FLAGS_HAS_DYNDATA = EVENT_FLAGS_SYSTEM_START,
+   EVENT_FLAGS_CNT
+};
+
+/** @brief Get event type flags value.
+ *
+ * @param flag Requested flag enum name.
+ * @param et   Pointer to the event type of the event that is processed by Event Manager.
```

Comment:
Just `Pointer to the event type`

Reply:
The event type does not need to be associated with a given event instance that is currently processed.